cmd/explain.go

```diff
@@ -0,0 +1,164 @@
+// Copyright © 2017 The Kubicorn Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"os"
+
+	"github.com/kris-nova/kubicorn/apis/cluster"
+	"github.com/kris-nova/kubicorn/cutil"
+	"github.com/kris-nova/kubicorn/cutil/initapi"
+
+	"github.com/kris-nova/kubicorn/cutil/logger"
+	"github.com/kris-nova/kubicorn/state"
+	"github.com/kris-nova/kubicorn/state/fs"
+	"github.com/kris-nova/kubicorn/state/git"
+	"github.com/kris-nova/kubicorn/state/jsonfs"
+	"github.com/spf13/cobra"
+	gg "github.com/tcnksm/go-gitconfig"
+)
+
+type ExplainOptions struct {
+	Options
+	Profile string
+	Output  string
+}
+
+type OutputData struct {
+	Actual   *cluster.Cluster
+	Expected *cluster.Cluster
+}
+
+var exo = &ExplainOptions{}
+
+// ListCmd represents the list command
```

Comment:
Looks like a  doc/code mismatch  on this function name.

Reply:
@nrb Fixed and rebased PR